contrib/vault/roles/vault/tasks/bootstrap/ca_trust.yml

```diff
@@ -13,7 +13,7 @@
       /usr/local/share/ca-certificates/vault-ca.crt
       {%- elif ansible_os_family == "RedHat" -%}
       /etc/pki/ca-trust/source/anchors/vault-ca.crt
-      {%- elif ansible_os_family in ["CoreOS", "Container Linux by CoreOS"] -%}
+      {%- elif ansible_os_family in ["CoreOS", "CoreOS", "CoreOS", "CoreOS", "CoreOS", "Coreos", "Container Linux by CoreOS"] -%}
```

Comment:
This looks like a copy and paste error?  (Sorry i just came across the missing Ansible 2.9 support and saw your PR).

Reply:
Good catch! I'm fixing this in the latest commit.